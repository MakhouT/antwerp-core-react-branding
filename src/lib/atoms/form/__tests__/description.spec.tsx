import * as React from "react";

import { renderAndMatchSnapshot } from "../../../../__tests__/__fixtures__/testUtils";
import Description from "../description";

describe("Description", () => {
    it("Renders default", () => {
        // Arrange
        const element = <Description
            name="description"
            description="blabla"
        />;

        // Act and assert
        renderAndMatchSnapshot(element);
    });
});
