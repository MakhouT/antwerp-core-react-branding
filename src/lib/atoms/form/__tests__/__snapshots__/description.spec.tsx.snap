// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Renders default 1`] = `
<small
  className=""
>
  blabla
</small>
`;
